packages/ubuntu_wsl_setup/windows/runner/main.cpp

```diff
@@ -26,7 +26,7 @@ int APIENTRY wWinMain(_In_ HINSTANCE instance, _In_opt_ HINSTANCE prev,
 
   FlutterWindow window(project);
   Win32Window::Point origin(10, 10);
-  Win32Window::Size size(1280, 720);
+  Win32Window::Size size(960, 680);

```

Comment:
- does this only change the default size? should there be any min/max/fixed size constraints because our slides won't scale to arbitrary sizes?
- could this be moved to a separate PR because we might want to include the respective formatting changes with the .clang-format config?

Reply:
Good point. That makes the size-related changes more interesting to put on a separate PR.